<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QMainWindow" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1283</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generador de Variables Aleatorias</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="left_panel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>731</height>
     </rect>
    </property>
    <property name="title">
     <string>Parámetros de Entrada</string>
    </property>
    <widget class="QPushButton" name="hist_btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Crear Histograma</string>
     </property>
    </widget>
    <widget class="QPushButton" name="generate_btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Generar Datos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="show_data_btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Mostrar Datos</string>
     </property>
    </widget>
    <widget class="QComboBox" name="dist_combo">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Uniforme [a, b]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Normal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Exponencial</string>
      </property>
     </item>
    </widget>
    <widget class="QSpinBox" name="size_input">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>141</width>
       <height>21</height>
      </size>
     </property>
     <property name="minimum">
      <number>1000</number>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="param1_input">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>90</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="param2_input">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>120</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QComboBox" name="bins_combo">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>150</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="close_btn">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>640</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Cerrar Programa</string>
     </property>
    </widget>
    <widget class="QLabel" name="dist_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Distribución:</string>
     </property>
    </widget>
    <widget class="QLabel" name="size_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Tamaño de muestra (&lt;= 1.000.000):</string>
     </property>
    </widget>
    <widget class="QLabel" name="param1_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Parámetro a:</string>
     </property>
    </widget>
    <widget class="QLabel" name="param2_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Parámetro b:</string>
     </property>
    </widget>
    <widget class="QLabel" name="bins_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Número de intervalos:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="right_panel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>851</width>
      <height>731</height>
     </rect>
    </property>
    <property name="title">
     <string>Resultados</string>
    </property>
    <widget class="QGraphicsView" name="right_layout">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>831</width>
       <height>511</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="data_display">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>550</y>
       <width>831</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
